src/net/chat.rs

```diff
@@ -1792,6 +1883,42 @@ impl ConversationP2P {
         Ok(None)
     }
 
+    /// Validate a pushed stackerdb chunk
+    /// Update bandwidth accounting, but forward the stackerdb chunk along.
+    /// Possibly return a reply handle for a NACK if we throttle the remote sender
+    fn validate_stackerdb_push(
+        &mut self,
+        local_peer: &LocalPeer,
+        chain_view: &BurnchainView,
+        preamble: &Preamble,
+        relayers: Vec<RelayData>,
+    ) -> Result<Option<ReplyHandleP2P>, net_error> {
+        assert!(preamble.payload_len > 1); // don't count 1-byte type prefix
+
+        if !self.process_relayers(local_peer, preamble, &relayers) {
+            debug!(
```

Comment:
Should this be a `warn!`? Would seeing this indicate something is wrong?

Reply:
Fixed